starters/tailwind/src/Link.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link as AriaLink, LinkProps as AriaLinkProps, composeRenderProps } from 'react-aria-components';
+import { tv } from 'tailwind-variants';
+import { focusRing } from './utils';
+
+interface LinkProps extends AriaLinkProps {
+  variant?: 'primary' | 'secondary'
+}
+
+const styles = tv({
+  extend: focusRing,
+  base: 'underline disabled:no-underline disabled:cursor-default forced-colors:disabled:text-[GrayText] transition rounded',
```

Comment:
Do we want to distinguish the primary/secondary links better from their disabled state? If you turn on isDisabled for either of the variants, all that changes is the underline disappears but the color remains the same. Do we want it to be grayed out?